server/document.js

```diff
@@ -97,11 +97,14 @@ export class NextScript extends Component {
 
   render () {
```

Comment:
Also, preload links would be helpful for any Link marked as "prefetch" since the browser can start those downloads right away. This possibility is now available since we are using plain JS (you can't preload JSON according to the spec)